Client.xcodeproj/xcshareddata/xcschemes/Fennec.xcscheme

```diff
@@ -140,8 +140,8 @@
    </BuildAction>
    <TestAction
       buildConfiguration = "Fennec"
-      selectedDebuggerIdentifier = ""
-      selectedLauncherIdentifier = "Xcode.IDEFoundation.Launcher.PosixSpawn"
+      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
```

Comment:
Curious wha this change is and if it was intentional

Reply:
This change is to be able to debug tests, I got annoyed that I couldn't put breakpoints in my unit tests.
 
It's this option here to debug executable, not sure why it was disabled:
![Screenshot 2023-09-15 at 11 24 57 AM](https://github.com/mozilla-mobile/firefox-ios/assets/11338480/00a9efb7-0509-41bb-8ef8-cf0e119e4f21)

Reply:
Good to know I had the same problem when I was doing the test for Redux